bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsAnalysis.scala

```diff
@@ -149,7 +149,7 @@ object AgsAnalysis {
    * Analysis of the given guide star in the given context, regardless of which
    * guide star is actually selected in the target environment.
    */
-  protected [ags] def analysis(ctx: ObsContext, mt: MagnitudeTable, guideProbe: ValidatableGuideProbe, guideStar: SiderealTarget, bands: BandsList): Option[AgsAnalysis] = {
+  def analysis(ctx: ObsContext, mt: MagnitudeTable, guideProbe: ValidatableGuideProbe, guideStar: SiderealTarget, bands: BandsList): Option[AgsAnalysis] = {
```

Comment:
I think this all looks quite plausible.  One detail that might be improved is that since now guide probes know about which bands are relevant we shouldn't need to pass the `BandsList` to this method.  Instead, it seems like it can just ask the `guideProbe` directly.